### PR TITLE
feat: config via GOOGLE_APPLICATION_CREDENTIALS

### PR DESCRIPTION
**Pull request by ekapratama93**
Add a way to load a credential file via the env variable with the path to the file. This should make it easier to configure like in GCP [example](https://cloud.google.com/docs/authentication/getting-started).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -107,6 +107,10 @@ if(process.env.GOOGLE_PROJECT_ID && process.env.GOOGLE_API_KEY){
         region: process.env.GOOGLE_DEFAULT_REGION || 'us-east1'
     };
 }
+if(process.env.GOOGLE_APPLICATION_CREDENTIALS){
+    GoogleConfig = require(process.env.GOOGLE_APPLICATION_CREDENTIALS)
+    GoogleConfig.project = GoogleConfig.project_id;
```
Comment (nazieb) [inline index.js:112]: where is the `GoogleConfig.project_id` set? the block if above?
if that's the case it's better to check the `GOOGLE_PROJECT_ID` env in this block and use it instead.

with current approach, you'll have to set GOOGLE_PROJECT_ID, GOOGLE_API_KEY & GOOGLE_APPLICATION_CREDENTIALS to get it properly configured

**Review by nazieb — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -107,6 +107,10 @@ if(process.env.GOOGLE_PROJECT_ID && process.env.GOOGLE_API_KEY){
...
+}
```
Comment (nazieb) [inline index.js:113]:
```suggestion
if(process.env.GOOGLE_PROJECT_ID && process.env.GOOGLE_APPLICATION_CREDENTIALS){
    GoogleConfig = require(process.env.GOOGLE_APPLICATION_CREDENTIALS)
    GoogleConfig.project = process.env.GOOGLE_PROJECT_ID;
}
```

File: index.js (hunk 1 above)
Comment (ekapratama93) [reply index.js:112]: The thing is, the service needs to load `project` in the internal logic but the default service JSON file from GCP only provides `project_id`. This change is to make that we only have to provide the file without the need to add another configuration. Please notice that it use separate `if` to check separate way to configure GoogleConfig

Comment (nazieb) [reply index.js:112]: ok, noted

**Review by nazieb — APPROVED** [review]

**Pull request merged**